deploy/playbooks/roles/celery/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: pip install celery
+  pip:
+    name: "celery[librabbitmq]"
```

Comment:
should we add this to setup.py instead?  I just thought of that when I posted this.